Chummer/data/gear.xml

```diff
@@ -1531,6 +1531,51 @@
         <damage>-1</damage>
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damage>6P Acid</damage>
+      </weaponbonus>
+      <ammoforweapontype>sling-shot</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>e20f021b-1d00-4443-a481-d417114e067e</id>
+      <name>Ammo: Sling-Shot Paint Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>10</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damage>0</damage>
```

Comment:
This will do nothing because `<damage>` just modifies the existing damage value of a weapon. So `<damage>0</damage>` means that the damage value is adjusted by 0, i.e. it remains unchanged.